<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>998</width>
    <height>583</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Canadian Epidemic Tracker</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show data for:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="e_s_selector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Epidemics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Supplies</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="type_selector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>H1N1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Small Pox</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HIV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Plague</string>
     </property>
    </item>
   </widget>
   <widget class="QWebView" name="mapView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Health Unit Information:</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="namebox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="idbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="numberof_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Cases:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="numberofbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="loadinglabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;Loading Health Unit Zones...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>510</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To move around the map and zoom, please use the navigation tool. Mouse drag &amp; zoom is not supported.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>23</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View Data</string>
    </property>
    <addaction name="actionTabular_Diseases_Data"/>
    <addaction name="actionSupplies_Data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CET"/>
   </widget>
   <widget class="QMenu" name="menuEnter_New_Data">
    <property name="title">
     <string>Add Data</string>
    </property>
    <addaction name="actionAdd_Disease_Data"/>
    <addaction name="actionAdd_Supplies_Data"/>
   </widget>
   <widget class="QMenu" name="menuShipments">
    <property name="title">
     <string>Shipping</string>
    </property>
    <addaction name="actionAdd_New_Shipment"/>
    <addaction name="actionManage_Shipments"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEnter_New_Data"/>
   <addaction name="menuView"/>
   <addaction name="menuShipments"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionAbout_CET">
   <property name="text">
    <string>About CET</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionShipments">
   <property name="text">
    <string>Tabular Disease Data</string>
   </property>
  </action>
  <action name="actionAdd_Disease_Data">
   <property name="text">
    <string>Add Case Report</string>
   </property>
  </action>
  <action name="actionAdd_Supplies_Data">
   <property name="text">
    <string>Add Inventory</string>
   </property>
  </action>
  <action name="actionAdd_New_Shipment">
   <property name="text">
    <string>Add New Shipment</string>
   </property>
  </action>
  <action name="actionManage_Shipments">
   <property name="text">
    <string>Manage Shipments</string>
   </property>
  </action>
  <action name="actionSupplies_Data">
   <property name="text">
    <string>View Inventory</string>
   </property>
  </action>
  <action name="actionShipments_2">
   <property name="text">
    <string>Shipments Manager</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTabular_Diseases_Data">
   <property name="text">
    <string>View Case Reports</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Disease_Data</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddDiseaseHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Supplies_Data</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddSupplyHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTabular_Diseases_Data</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewDiseasesHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSupplies_Data</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>ViewSuppliesHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_New_Shipment</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddShipmentHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManage_Shipments</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>ManageShipmentsHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_CET</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>AboutCETHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>EditDataHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddDiseaseHandler()</slot>
  <slot>AddSupplyHandler()</slot>
  <slot>ViewDiseasesHandler()</slot>
  <slot>ViewSuppliesHandler()</slot>
  <slot>AddShipmentHandler()</slot>
  <slot>ManageShipmentsHandler()</slot>
  <slot>AboutCETHandler()</slot>
  <slot>EditDataHandler()</slot>
 </slots>
</ui>
